### DISCUSSION TITLE
Several suggestions for the UI order window

### DISCUSSION THREAD

**Discussion by ArtemV007** [Chia-Network/chia-blockchain-gui#2375] [Ideas]
1)  I want to suggest adding various filters and sorting for tables. Over time, it becomes difficult to find what you need
![Screenshot from 2023-12-22 23-54-44](https://github.com/Chia-Network/chia-blockchain/assets/45736498/bc41b141-e730-4873-96fc-6d7eb0cf03af)

2) would it be better to move the Create Offer and View Offer buttons to the upper-right corner to save space? at least on desktop it could work well. 
<img width="1762" alt="Group 23" src="https://github.com/Chia-Network/chia-blockchain/assets/45736498/50c2cadf-ed57-4f8f-81e0-a3b75f8c325b">

3) Perhaps it was a good idea to be able to hide current tables

Just my thoughts :)

**Comment by ArtemV007** [comment]
4) I would also like to point out such a problem. When we specify a commission, it does not reflect mempool downloads. It so happened that I chose the recommended commission and have been waiting for 2 days, although the selection mentioned a minute. As I understand it, clients cannot yet receive information in real time and analyze it to prompt the user.

Maybe from the point of view of UX add an icon with text that you can check the network load yourself https://dashboard.chia.net/d/46EAA05E/mempool-transactions-and-fees?orgId=1. I think that would be great.
![Screenshot from 2023-12-23 23-42-18](https://github.com/Chia-Network/chia-blockchain/assets/45736498/39c028bb-0638-4869-b6b7-66897dd7cde2)

**Comment by BrandtH22** [comment]
Hey @ArtmenV , thank you very much for the recommendations. 

Our team is actively working on improvements to the fee estimator to ensure it becomes more useful and accurate.
We are also looking into a few different display options to make the coinset model of chia easier and more intuitive to work with while provided additional view options for the user to filter their coins. 

We will keep this discussion open while working through these updates and feel free to provide any other feedback regarding here, thank you!

**Comment by BrandtH22** [comment]
@ArtmenV  , note I transferred this enhancement request to the GUI repo for better visibility 👍
